middleware/helpers.js

```diff
@@ -50,3 +50,35 @@ module.exports.getMaxDate = () => {
 
   return `${year}-${month}-${day}`;
 };
+
+function checkMediaLink(mediaLink, linkRegex) {
+  const match = mediaLink.match(linkRegex);
+  return match && match[3];
+}
+
+module.exports.mediaLinkValidation = (facebook, twitter, instagram, tumblr) => {
+  const mediaRegex = {
+    facebook:
```

Comment:
I think there's no need to store regex patterns in an object, it's not used outside this scope anyway.

```
const facebookRegex = ...
```